<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>ABACBS/BiocAsia Workshop Schedule 2025</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="index_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-1e1b6d7d3018d0281420745e134191b0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<style>

      .quarto-title-block .quarto-title-banner {
        background: #0d1413;
      }
</style>
<script src="index_files/libs/core-js-2.5.3/shim.min.js"></script>
<script src="index_files/libs/react-18.2.0/react.min.js"></script>
<script src="index_files/libs/react-18.2.0/react-dom.min.js"></script>
<script src="index_files/libs/reactwidget-2.0.0/react-tools.js"></script>
<link href="index_files/libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="index_files/libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="index_files/libs/reactable-0.4.4/reactable.css" rel="stylesheet">
<script src="index_files/libs/reactable-binding-0.4.4/reactable.js"></script>


</head>

<body class="quarto-light">

<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-page">
      <h1 class="title">ABACBS/BiocAsia Workshop Schedule 2025</h1>
                      </div>
  </div>
    
  
  <div class="quarto-title-meta column-page">

      
    
      
    </div>
    
  
  </header><div id="quarto-content" class="page-columns page-rows-contents page-layout-full">

<main class="content quarto-banner-title-block column-page" id="quarto-document-content">





<p><img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/421bf838-6118-4ae6-b7d9-b556072d4524/ABACBS+2025+Banner+-+10+years+of+ABACBS.png?format=2500w" class="img-fluid"></p>
<div class="cell">
<style type="text/css">
.workshop-detail {
  padding: 16px;
  background-color: #f9f9f9;
  border-radius: 8px;
  margin-top: 8px;
  text-align: left;
}
table {
  border-collapse: collapse;
  width: 100%;
}
td, th {
  border: 1px solid black;
  padding: 8px;
  text-align: center;
  font-size: 14px;
}
.quarto-title-block .quarto-title-banner {
  background-image: url(https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/1502173845208-XSB852DA7HSNFNFFIMEA/ABACBS+logo+folded+white.png);
  background-size: 300px;
  background-position: left;
  background-repeat: no-repeat;
  padding-left: 20px;
  background-origin: content-box;
  text-align: center;
}
</style>
</div>
<p>The ABACBS25 Conference have joined with BiocAsia to organise two full days of workshops immediately following the main conference. All workshops will be held in <a href="https://maps.app.goo.gl/gwrbmTE6J4puUPum9">Festival Tower</a> overlooking the Festival Theatre and the River Torrens.</p>
<p>All are welcome, including those who have attended the ABACBS Conference and those who simply wish to attend some high-quality training workshops. Registrations <em>are now open</em> on the main <a href="https://www.abacbs.org/abacbs-2025-workshops">ABACBS workshop website</a></p>
<p>Catering will be provided on both days on Level 5</p>
<ul>
<li>10:30-11:00am Morning Tea</li>
<li>12:30-1:30pm Lunch</li>
<li>3:00-3:30pm Afternoon Tea</li>
</ul>
<p>Please find an outline of the currently scheduled sessions immediately below, along with a <a href="#full-table-with-abstracts">browsable table</a> including all abstracts</p>
<p>Setup instructions for select workshops are also <a href="info.html">available here</a></p>
<section id="complete-workshop-schedule" class="level2">
<h2 class="anchored" data-anchor-id="complete-workshop-schedule">Complete Workshop Schedule</h2>
<section id="thursday-27th-november" class="level3">
<h3 class="anchored" data-anchor-id="thursday-27th-november">Thursday 27th November</h3>
<table>
<tbody><tr>
<th>
</th>
<th colspan="3" style="background-color: #b6d0e2;">
ABACBS
</th>
<th colspan="2" style="background-color: #e2c8b6;">
BiocAsia
</th>
</tr>
<tr>
<th>
Time
</th>
<th style="background-color: #b6bae2;">
Professional Bioinformatics<br>(Room 306)
</th>
<th style="background-color: #bae2b6;">
Long Reads<br>(Room 310)
</th>
<th style="background-color: #b6e2de;">
Scalable Workflows &amp; Compute Infrastructure<br>(Room 415)
</th>
<th style="background-color: #e2deb6;">
Microbiome &amp; Metagenomics<br>(Room 605)
</th>
<th style="background-color: #e2b6ba;">
Single-Cell and Spatial Transcriptomics<br>(Room 506)
</th>
</tr>
<tr>
<th>
8:30
</th>
<th colspan="5">
Welcome to The ABACBS/BiocAsia Workshops (Room 605)
</th>
</tr>
<tr>
<th>
9:00
</th>
<td rowspan="2" style="background-color: #b6bae2;">
Introduction to Agentic Development
</td>
<td style="background-color: #bae2b6;">
Introduction to Oxford Nanopore
</td>
<td rowspan="2" style="background-color: #b6e2de;">
Leveraging peak Australian compute to enable workflows for predictive structural biology at scale
</td>
<td rowspan="2" style="background-color: #e2deb6;">
Orchestrating Microbiome Analysis with Bioconductor
</td>
<td rowspan="2" style="background-color: #e2b6ba;">
Unlocking single-cell spatial omics analyses with SCDNEY: Characterising spatial relationships between cells
</td>
</tr>
<tr>
<th>
11:00
</th>
<td style="background-color: #bae2b6;">
Using long reads to analyse genetic variation
</td>
</tr>
<tr>
<th>
12:30
</th>
<th colspan="5">
Lunch
</th>
</tr>
<tr>
<th>
1:30
</th>
<td rowspan="2">
</td>
<td style="background-color: #e2b6ba;">
<strong>Single-Cell and Spatial Transcriptomics<br></strong>DCS Cloud: Revolutionizing Genomics Research through Integrated, AI-Driven Solutions
</td>
<td rowspan="2" style="background-color: #b6e2de;">
Client-side bioinformatics: building robust bioinformatics tools for the browser with WebAssembly
</td>
<td rowspan="2" style="background-color: #e2deb6;">
Making Sense of Metagenomes
</td>
<td style="background-color: #e2b6ba;">
<b>Room 505</b><br>Streamline spatial and transcriptomic analysis with tidyomics
</td>
</tr>
<tr>
<th>
3:30
</th>
<td style="background-color: #e2b6ba;">
A taster of in situ spatial transcriptomics experimental analysis
</td>
<td rowspan="2" style="background-color: #e2b6ba;">
<b>Room 505</b><br>Spatial clustering with ClustSIGNAL – a spatial transcriptomics data analysis method
</td>
</tr>
</tbody></table>
</section>
<section id="friday-28th-november" class="level3">
<h3 class="anchored" data-anchor-id="friday-28th-november">Friday 28th November</h3>
<table>
<tbody><tr>
<th>
</th>
<th style="background-color: #b6d0e2;">
ABACBS
</th>
<th colspan="2" style="background-color: #e2c8b6;">
BiocAsia
</th>
</tr>
<tr>
<th>
Time
</th>
<th style="background-color: #b6d0e2;">
AI and Machine Learning in Genomics<br>(Room 309)
</th>
<th style="background-color: #e2deb6;">
Single Cell Omics<br>(Room 506)
</th>
<th style="background-color: #e2b6ba;">
Spatial Transcriptomics<br>(Room 505)
</th>
</tr>
<tr>
<th>
9:00
</th>
<td style="background-color: #b6d0e2;">
Decoding Biological Data with Random Matrix Theory: Applications in Single-Cell Transcriptomics and Viral Vaccine Design
</td>
<td style="background-color: #e2deb6;">
ATAClone: cancer clone identification and copy number estimation from single-cell ATAC-seq
</td>
<td rowspan="2" style="background-color: #e2b6ba;">
Spatial transcriptomics normalisation and phenotyping using SpaNorm, PhiSpace and jazzPanda
</td>
</tr>
<tr>
<th>
11:00
</th>
<td style="background-color: #b6d0e2;">
CANCELLED<br><s>Identifying Biomarkers of Drug Sensitivity and Predicting Treatment Responsive Colon Cancer Patients</s>
</td>
<td style="background-color: #e2deb6;">
Advanced Marker Identification and Data Analysis with mastR and smartid
</td>
</tr>
<tr>
<th>
12:30
</th>
<th colspan="5">
Lunch
</th>
</tr>
<tr>
<th>
1:30
</th>
<td style="background-color: #b6d0e2;">
CANCELLED<br><s>Identifying Biomarkers of Drug Sensitivity and Predicting Treatment Responsive Colon Cancer Patients</s>
</td>
<td style="background-color: #b6e2de;">
<strong>Scalable Workflows &amp; Compute Infrastructure<br></strong>Interpreting benchmarking results with BenchmarkInsights
</td>
<td style="background-color: #e2b6ba;">
Interrogating spatial transcriptomics: hands-on spatial domain, neighborhood &amp; density analysis with DOMINO, hoodscanR and scider
</td>
</tr>
<tr>
<th>
3:00
</th>
<th colspan="5">
Closing Comments (Room 505)
</th>
</tr>
<tr>
<th>
3:30
</th>
<th colspan="5">
Afternoon Tea
</th>
</tr>
</tbody></table>
</section>
</section>
<section id="full-table-with-abstracts" class="level2">
<h2 class="anchored" data-anchor-id="full-table-with-abstracts">Full Table With Abstracts</h2>
<div id="tbl-workshop-schedule" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl quarto-uncaptioned" id="tbl-workshop-schedule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1
</figcaption>
<div aria-describedby="tbl-workshop-schedule-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">

<em>
    Full table showing all workshops. All columns are searchable.
    Please click the arrows at the left to show the complete abstract as 
    submitted with a full description of the session
    </em>

<div class="reactable html-widget html-fill-item anchored" id="tbl-workshop-schedule" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="tbl-workshop-schedule">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Day":["Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Thursday","Friday","Friday","Friday","Friday","Friday","Friday"],"Start":["2025-11-27T09:00:00Z","2025-11-27T09:00:00Z","2025-11-27T09:00:00Z","2025-11-27T09:00:00Z","2025-11-27T09:00:00Z","2025-11-27T11:00:00Z","2025-11-27T13:30:00Z","2025-11-27T13:30:00Z","2025-11-27T13:30:00Z","2025-11-27T13:30:00Z","2025-11-27T15:30:00Z","2025-11-27T15:30:00Z","2025-11-28T09:00:00Z","2025-11-28T09:00:00Z","2025-11-28T09:00:00Z","2025-11-28T11:00:00Z","2025-11-28T13:30:00Z","2025-11-28T13:30:00Z"],"End":["2025-11-27T12:30:00Z","2025-11-27T12:30:00Z","2025-11-27T12:30:00Z","2025-11-27T12:30:00Z","2025-11-27T10:30:00Z","2025-11-27T12:30:00Z","2025-11-27T17:00:00Z","2025-11-27T17:00:00Z","2025-11-27T15:00:00Z","2025-11-27T15:00:00Z","2025-11-27T17:00:00Z","2025-11-27T17:00:00Z","2025-11-28T12:30:00Z","2025-11-28T10:30:00Z","2025-11-28T10:30:00Z","2025-11-28T12:30:00Z","2025-11-28T15:00:00Z","2025-11-28T15:00:00Z"],"Domain":["ABACBS","ABACBS","BiocAsia","BiocAsia","ABACBS","ABACBS","ABACBS","BiocAsia","ABACBS","BiocAsia","ABACBS","BiocAsia","BiocAsia","ABACBS","BiocAsia","BiocAsia","BiocAsia","BiocAsia"],"Stream":[["Professional Bioinformatics","AI and Machine Learning in Genomics","Production Deployments"],"Scalable Workflows & Compute Infrastructure","Metagenomics & Microbiome analyses","Spatial Transcriptomics","Long Read Technology","Long Read Technology",["Scalable Workflows & Compute Infrastructure","Professional Bioinformatics"],"Metagenomics & Microbiome analyses",["Single-Cell Omics","Spatial Transcriptomics","AI and Machine Learning in Genomics"],"Spatial Transcriptomics","Spatial Transcriptomics","Spatial Transcriptomics","Spatial Transcriptomics",["Single-Cell Omics","AI and Machine Learning in Genomics"],"Single-Cell Omics","Single-Cell Omics","Scalable Workflows & Compute Infrastructure","Spatial Transcriptomics"],"Title":["Introduction to agentic development","Leveraging peak Australian compute to enable workflows for predictive structural biology at scale","Orchestrating Microbiome Analysis with Bioconductor","Unlocking single-cell spatial omics analyses with SCDNEY: Characterising spatial relationships between cells","Introduction to Nanopore","Using long reads to analyse genetic variation","Client-side bioinformatics: building robust  bioinformatics tools for the browser with  WebAssembly","Making Sense of Metagenomes","DCS Cloud: Revolutionizing Genomics Research through Integrated, AI-Driven Solutions","Streamline spatial and transcriptomic analysis with tidyomics","A taster of in situ spatial transcriptomics experimental analysis","Spatial clustering with ClustSIGNAL – a spatial transcriptomics data analysis method","Introduction to spatial transcriptomics data analysis","Decoding Biological Data with Random Matrix Theory: Applications in Single-Cell Transcriptomics and Viral Vaccine Design","ATAClone: cancer clone identification and copy number estimation from single-cell ATAC-seq","Advanced Marker Identification and Data Analysis with mastR and smartid","Interpreting benchmarking results with BenchmarkInsights","Interrogating spatial transcriptomics: hands-on spatial domain, neighborhood & density analysis with DOMINO, hooscanR and scider."],"Presenter":[["Raisa Hashem","Cain Hopwood","Emma Arrigo"],["Johan Gustafsson","Ziad Al Bkhetan","Thomas Litfin","Sarah Beecroft","Georgie Samaha","Farah Zaib Khan"],["Tuomas Borman","Leo Lahti"],["Daniel Kim,,","Farhan Ameen,,","Lijia Yu,","Shreya Rajesh Rao,,","Ellis Patrick,,,","Jean Yang,,"],["Andre Sim"],["Andre Sim"],["Wytamma Wirth,","Andrew Lonsdale,,","Leo Featherstone","Torsten Seemann,"],["Feargal Ryan","Calum Walsh","Geraldine Kong","Vini Salazar,","Don Teng"],["Karthik Pullela","Mingyan Fang","Melvin Xiaochun Li"],["William Hutchison,"],["Sarah Williams","Magda Antczak","Nick Matigan"],["Pratibha Panwar,,","Shila Ghazanfar,,"],["Dharmesh Bhuva","Jiadong Mao","Melody Jin"],["Syed Faraz Ahmed","Ahmed Abdul Quadeer","Raymond Louie"],["Lachlan Cain","Sanduni Rajapaksa"],["Jinjin Chen","Chin Wee Tan","Liyang Fei"],["Xiaoqi Cabiria Liang,,","Dario Strbenac,,","Yue Cao,,","Jean Yang,,"],["Ning Liu","Nora Liu","Monika Mohenska","Jose Polo"]],"Institution":["Amazon Web Services",["Australian BioCommons","University of Melbourne","University of New South Wales","Pawsey Supercomputing Research Centre","Sydney Informatics Hub","University of Sydney"],["University of Turku","Finland"],["Sydney Precision Data Science Centre","The University of Sydney","NSW","Australia","Computational Systems Biology Unit","Children’s Medical Research Institute","Westmead","Charles Perkins Centre","School of Medical Sciences","Faculty of Medicine and Health","School of Mathematics and Statistics"],"Oxford Nanopore Technologies","Oxford Nanopore Technologies",["Department of Microbiology and Immunology","University of Melbourne","The Peter Doherty Institute for Infection and Immunity","Victoria","Australia","Centre for Pathogen Genomics","Peter MacCallum Cancer Centre","Melbourne","3000","Sir Peter MacCallum Department of Oncology","The University of Melbourne"," Parkville","3010","Murdoch Children’s Research Institute","Parkville","3052","Kirby Institute","UNSW","Sydney"],["Flinders University","University of Melbourne","Melbourne Bioinformatics","Metabolomics SA"],"MGI",["Walter and Eliza Hall Institute of Medical Research","Parkville","Australia","Department of Medical Biology","University of Melbourne"],"QCIF",["School of Mathematics and Statistics","University of Sydney","Sydney Precision Data Science Centre","Charles Perkins Centre"],["Frazer Institute","University of Queensland","Melbourne Integrative Genomics (MIG)","University of Melbourne","Walter and Eliza Hall Institute of Medical Research"],["Department of Electrical and Electronic Engineering","The University of Melbourne","School of Computer Science and Engineering","University of New South Wales"],"Peter MacCallum Cancer Centre",["Bioinformatics Division of Walter and Eliza Hall Institute of Medical Research","Peter MacCallum Cancer Centre"],["School of Mathematics and Statistics","The University of Sydney","NSW 2006","Australia.","Sydney Precision Data Science Centre","Australia","Charles Perkins Centre"],["Adelaide Centre for Epigenetics","University of Adelaide"]]},"columns":[{"id":".details","name":"","type":null,"sortable":false,"resizable":false,"filterable":false,"searchable":false,"width":45,"align":"center","details":[{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Introduction to agentic development"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Raisa Hashem",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Cain Hopwood",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Emma Arrigo",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Amazon Web Services"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 09:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n      Agentic Workflows enable large language models (LLMs) to act autonomously, \n      performing tasks or providing assistance on behalf of users. This mini \n      hackathon will introduce the participants to agentic development techniques. \n      They’ll have access to industry leading tools that accelerate development in \n      the hands-on lab. The facilitators will demonstrate how to get the most out \n      of this emerging tech, then provide guidance as participants build their own \n      application. Come with your own blue-sky idea, and bring it to life. \n      Or just work through the guided examples. The choice is yours."]},{"name":"p","attribs":{},"children":[{"name":"strong","attribs":{},"children":["Who should attend"]},": Engineers, architects, and scientists who want to see what \n      this whole “vibe coding” thing is about (and try it for themselves)."]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Professional Bioinformatics"]},{"name":"li","attribs":{},"children":["AI and Machine Learning in Genomics"]},{"name":"li","attribs":{},"children":["Production Deployments"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Leveraging peak Australian compute to enable workflows for predictive structural biology at scale"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Johan Gustafsson",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Ziad Al Bkhetan",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Thomas Litfin",{"name":"sup","attribs":{},"children":["2"]}]},{"name":"Fragment","attribs":[],"children":[", ","Sarah Beecroft",{"name":"sup","attribs":{},"children":["3"]}]},{"name":"Fragment","attribs":[],"children":[", ","Georgie Samaha",{"name":"sup","attribs":{},"children":["4"]}]},{"name":"Fragment","attribs":[],"children":[", ","Farah Zaib Khan",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Australian BioCommons, University of Melbourne"]},{"name":"li","attribs":{},"children":["University of New South Wales, Australian BioCommons"]},{"name":"li","attribs":{},"children":["Pawsey Supercomputing Research Centre"]},{"name":"li","attribs":{},"children":["Sydney Informatics Hub, University of Sydney"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 09:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":["\n      Deep learning for structural biology has numerous applications, ranging \n      from basic research through to clinical application. A key challenge with \n      these approaches is accessibility, and much work is currently being \n      conducted to develop computational workflows to improve this access. \n      However, building, executing, and sharing workflows is challenging. The \n      difficulty only increases when implementing novel approaches like deep \n      learning, and even more so when moving these workflows to national\n      infrastructure to operate at scale. Some of the challenges include knowing \n      how to develop a workflow to optimise use of these large compute systems, \n      how to configure and deploy workflows that can efficiently employ both CPU \n      and GPU, and how to share the workflows created so that others can find,\n      understand, reuse, and ultimately cite the workflow. This workshop will \n      examine these challenges from the perspective of a workflow’s life cycle, \n      and in doing so provide both theory and hands-on activities that will \n      support bioinformaticians to develop, execute, and share computational \n      workflows for structural biology at scale.\n      "]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Scalable Workflows & Compute Infrastructure"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Orchestrating Microbiome Analysis with Bioconductor"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Tuomas Borman",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Leo Lahti",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["University of Turku, Finland"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 09:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n        In microbiome research, there is an increasing demand for robust,\n        transparent, and reproducible computational methods. Bioconductor is\n        well-positioned to to fulfill this need by providing high-quality, open-source\n        bioinformatics software. At the core of Bioconductor are standardized data\n        containers that harmonize data management and enable interoperability across\n        the ecosystem.\n        "]},{"name":"p","attribs":{},"children":["\n        This workshop introduces the latest advances in microbiome analysis within\n        Bioconductor, focusing on the mia (Microbiome Analysis) framework. The framework\n        is built upon the (Tree)SummarizedExperiment data container — the central data\n        structure in Bioconductor — and provides a comprehensive suite of statistical\n        tools for microbiome data science. It is continuously expanding with new methods\n        and offers native support for multi-table and hierarchical data, while being\n        computationally more efficient than previous approaches. These features enable\n        users to build modular and reproducible workflows for microbiome analysis.\n        "]},{"name":"p","attribs":{},"children":["\n        During the workshop, participants will gain hands-on experience with the\n        (Tree)SummarizedExperiment data container and the mia framework through a\n        practical case study. The practicals cover common downstream analysis steps,\n        from data import and wrangling to diversity estimation, differential abundance\n        analysis, and visualization. After the workshop, participants will have the\n        skills to continue learning through the freely available online book\n        Orchestrating Microbiome Analysis with Bioconductor\n        (https://microbiome.github.io/OMA), which promotes best practices and\n        supports adoption of the Bioconductor ecosystem. Together, these resources\n        foster scalable, transparent, and community-driven microbiome data science.\n        "]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Metagenomics & Microbiome analyses"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Unlocking single-cell spatial omics analyses with SCDNEY: Characterising spatial relationships between cells"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Daniel Kim",{"name":"sup","attribs":{},"children":["1,2,4"]}]},{"name":"Fragment","attribs":[],"children":[", ","Farhan Ameen",{"name":"sup","attribs":{},"children":["1,3,5"]}]},{"name":"Fragment","attribs":[],"children":[", ","Lijia Yu",{"name":"sup","attribs":{},"children":["1,5"]}]},{"name":"Fragment","attribs":[],"children":[", ","Shreya Rajesh Rao",{"name":"sup","attribs":{},"children":["1,3,5"]}]},{"name":"Fragment","attribs":[],"children":[", ","Ellis Patrick",{"name":"sup","attribs":{},"children":["1,2,3,5"]}]},{"name":"Fragment","attribs":[],"children":[", ","Jean Yang",{"name":"sup","attribs":{},"children":["1,3,5"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Sydney Precision Data Science Centre, The University of Sydney, NSW, Australia"]},{"name":"li","attribs":{},"children":["Computational Systems Biology Unit, Children’s Medical Research Institute, Westmead, NSW, Australia"]},{"name":"li","attribs":{},"children":["Charles Perkins Centre, The University of Sydney, NSW, Australia"]},{"name":"li","attribs":{},"children":["School of Medical Sciences, Faculty of Medicine and Health, The University of Sydney, NSW, Australia"]},{"name":"li","attribs":{},"children":["School of Mathematics and Statistics, The University of Sydney, NSW, Australia"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 09:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n        Spatial omics technologies such as IMC, PCF, Xenium and CosMx are revolutionising our understanding of\n        cellular organisation and function. These platforms generate complex datasets that can uncover disease\n        relevant biology but only if the data are analysed and interpreted appropriately. Each step of the analysis\n        pipeline involves key decisions, from segmentation strategies to cell type annotation and spatial statistics.\n        These choices can strongly influence biological conclusions, yet they are often made without fully\n        understanding their assumptions, limitations or alternatives.\n        "]},{"name":"p","attribs":{},"children":["\n        In this workshop we will introduce an analytical framework built around our SCDNEY suite of Bioconductor\n        packages, many of which are already widely used in the ABACBS community. We will demonstrate how\n        simpleSeg, scHOT, FuseSOM, scClassify, spicyR, lisaClust, statial, scFeatures and ClassifyR can be combined into\n        flexible pipelines. Beyond simply teaching how to run the tools, we will guide participants to critically\n        evaluate:\n        "]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Initial exploration of spatial data, including quality assessment driven by questions such as:",{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Does the study contain sufficient information for downstream analysis?"]},{"name":"li","attribs":{},"children":["Are there cells or samples that should be filtered out or down-weighted?"]},{"name":"li","attribs":{},"children":["Which segmentation approaches are best suited for their data and why?"]}]}]},{"name":"li","attribs":{},"children":["How normalisation and cell type identification choices can bias downstream analyses"]},{"name":"li","attribs":{},"children":["What constitutes robust evidence of cell–cell interactions or microenvironmental effects"]},{"name":"li","attribs":{},"children":["How to select appropriate spatial statistical tests and interpret their outputs"]},{"name":"li","attribs":{},"children":["Identification of cohort heterogeneity and its implications via ClassifyR. Here we address:",{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["What modalities are most informative for which subcohort ?"]},{"name":"li","attribs":{},"children":["What are the characteristics of subcohorts that are separated by predictability ?"]}]}]}]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Introduction to Nanopore"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Andre Sim",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Oxford Nanopore Technologies"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 09:00AM"," to ","10:30AM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Intro to Nanopore data and Epi2me"]},{"name":"li","attribs":{},"children":["What are the data types"]},{"name":"li","attribs":{},"children":["Dorado + rebasecalling, demux (CPU can be used for dorado rebasecalling \n        for a small number of reads, we will provide raw data to basecall and demux)"]},{"name":"li","attribs":{},"children":["What is a squiggle?"]},{"name":"li","attribs":{},"children":["Event alignment, and visualisation of raw signal data"]},{"name":"li","attribs":{},"children":["How to quality control long-read data. Provide data to review and QC"]}]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Long Read Technology"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Using long reads to analyse genetic variation"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Andre Sim",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Oxford Nanopore Technologies"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 11:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["Attendees are expected to have attended the Introduction to Nanopore workshop run immediately preceding this one"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Participants to run subsampled human data to run through wf-human-variation via command line"]},{"name":"li","attribs":{},"children":["Discussion on individual workflow components and how and which can be applied to different organisms"]},{"name":"li","attribs":{},"children":["We will share output of the workflow from a full human dataset for downstream analysis"]},{"name":"li","attribs":{},"children":["How to find variants of interest in vcf, SVs, CNVs via the report and IGV"]},{"name":"li","attribs":{},"children":["Visualise genomic methylation"]}]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Long Read Technology"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Client-side bioinformatics: building robust  bioinformatics tools for the browser with  WebAssembly"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Wytamma Wirth",{"name":"sup","attribs":{},"children":["1,2"]}]},{"name":"Fragment","attribs":[],"children":[", ","Andrew Lonsdale",{"name":"sup","attribs":{},"children":["3,4,5"]}]},{"name":"Fragment","attribs":[],"children":[", ","Leo Featherstone",{"name":"sup","attribs":{},"children":["6"]}]},{"name":"Fragment","attribs":[],"children":[", ","Torsten Seemann",{"name":"sup","attribs":{},"children":["1,2"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Department of Microbiology and Immunology, University of Melbourne, The Peter Doherty Institute for Infection and Immunity, Victoria, Australia"]},{"name":"li","attribs":{},"children":["Centre for Pathogen Genomics, University of Melbourne, Victoria, Australia"]},{"name":"li","attribs":{},"children":["Peter MacCallum Cancer Centre, Melbourne, 3000, Australia"]},{"name":"li","attribs":{},"children":["Sir Peter MacCallum Department of Oncology, The University of Melbourne,  Parkville, 3010, Australia"]},{"name":"li","attribs":{},"children":["Murdoch Children’s Research Institute, Parkville, 3052, Australia"]},{"name":"li","attribs":{},"children":["Kirby Institute, UNSW, Sydney"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 01:30PM"," to ","05:00PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n        Web servers are a common way to deploy bioinformatics software to relieve \n        end users of  installation issues. Unfortunately, these servers are \n        frequently unmaintained a few years  after publication, and they may sit \n        idle most of the time – if they are still available at all. Sending data \n        to external servers can also raise confidentiality concerns for clinical \n        or sensitive datasets.\n        "]},{"name":"p","attribs":{},"children":["\n        WebAssembly (Wasm) offers solutions to these \n        issues by running computations client‑side in the browser. Wasm is now \n        mature in the web ecosystem and is  gaining traction in bioinformatics. \n        In this workshop we will provide an overview of Wasm  and the surrounding \n        tooling, with examples of real‑world bioinformatics deployments. In the \n        second half, participants will build and deploy a complete bioinformatics \n        Wasm application that delivers state‑of‑the‑art analyses in the browser \n        with zero server costs.  We will also cover practical limits and \n        trade‑offs, including performance, memory,  threading, GPU acceleration, \n        data handling, and hosting options.\n        "]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Scalable Workflows & Compute Infrastructure"]},{"name":"li","attribs":{},"children":["Professional Bioinformatics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Making Sense of Metagenomes"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Feargal Ryan",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Calum Walsh",{"name":"sup","attribs":{},"children":["2"]}]},{"name":"Fragment","attribs":[],"children":[", ","Geraldine Kong",{"name":"sup","attribs":{},"children":["2"]}]},{"name":"Fragment","attribs":[],"children":[", ","Vini Salazar",{"name":"sup","attribs":{},"children":["2,3"]}]},{"name":"Fragment","attribs":[],"children":[", ","Don Teng",{"name":"sup","attribs":{},"children":["4"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Flinders University"]},{"name":"li","attribs":{},"children":["University of Melbourne"]},{"name":"li","attribs":{},"children":["Melbourne Bioinformatics"]},{"name":"li","attribs":{},"children":["Metabolomics SA"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 01:30PM"," to ","05:00PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n      This workshop will provide a hands-on introduction to current best \n      practices in metagenomic data analysis. Attendees will be guided from \n      theory through to analysis, visualisation, and biological interpretation \n      using public datasets and R Markdown notebooks. The workshop is divided \n      into four structured, practical blocks:\n      "]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":[{"name":"strong","attribs":{},"children":["Block 1 – Introduction to Metagenomic Analyses: "]},"A theoretical session covering core concepts, the history of metagenomics, key methodologies, and landmark studies in the field."]},{"name":"li","attribs":{},"children":[{"name":"strong","attribs":{},"children":["Block 2 – Analysing Metagenomic Data with Nextflow and nf-core: "]},"Hands-on session running two nf-core pipelines—mag and taxprofiler—for metagenome assembly and profiling."]},{"name":"li","attribs":{},"children":[{"name":"strong","attribs":{},"children":["Block 3 – Data Exploration and Interpretation with Bioconductor: "]},"Practical session using pre-computed outputs in R to explore and visualise metagenomic datasets"]},{"name":"li","attribs":{},"children":[{"name":"strong","attribs":{},"children":["Block 4 – Integrating Metagenomics and Metabolomics: "]},"Hands-on analysis of pre-computed multi-omics data in R, with an introduction to metabolomics and integrative methods."]}]},{"name":"p","attribs":{},"children":["\n      This workshop brings together instructors from South Australia and \n      Victoria with expertise spanning microbiome analysis, metabolomics, and \n      bioinformatics education. All materials will be developed using Melbourne \n      Bioinformatics training templates and made publicly available post-workshop.\n      "]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Metagenomics & Microbiome analyses"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["DCS Cloud: Revolutionizing Genomics Research through Integrated, AI-Driven Solutions"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Karthik Pullela",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Mingyan Fang",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Melvin Xiaochun Li",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["MGI"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 01:30PM"," to ","03:00PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"h3","attribs":{},"children":["Theme:"]},{"name":"p","attribs":{},"children":["This workshop directly aligns with the conference theme of Single-Cell & \n      Spatial Omics, with a focus on novel analysis techniques, streamlined \n      workflows, and practical approaches for interpreting multi-omics data \n      including spatiotemporal omics, single-cell and large-scale human cohorts."]},{"name":"h3","attribs":{},"children":["Abstract:"]},{"name":"p","attribs":{},"children":["As the complexity and volume of omics data explode, researchers face \n      increasingly severe bottlenecks in data processing, cross-team \n      collaboration, and analytical reproducibility. This workshop introduces ",{"name":"strong","attribs":{},"children":["DCS Cloud (DNA, Cell Omics and SpatialOmics)"]}," an AI-driven, cloud-based platform built to overcome these challenges and\n      democratize access to advanced analysis pipelines for genomics, single-cell,\n      and spatial omics research. Through a combination of expert talks and a \n      practical training, attendees will gain a comprehensive understanding of \n      how DCS Cloud’s integrated framework (featuring automated pipelines, \n      collaborative environments, and high-resolution spatial visualization \n      tools) can transform their research productivity."]},{"name":"h3","attribs":{},"children":["Workshop Agenda and Speakers:"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Introduction to DCS Cloud and Stereo-seq (20 min) ",{"name":"em","attribs":{},"children":["Dr. Karthik Pullela, Application Scientist, MGI Australia "]},"Overview of the platform’s full-stack architecture, its applications in\n        cohort studies and brain science, and key spatial technologies such as \n        Stereoseq and StereoMap"]},{"name":"li","attribs":{},"children":["Applied User Cases in Population Genomics and 3D Spatial Brain Mapping (30 min) ",{"name":"em","attribs":{},"children":["Dr. Mingyan Fang, Chief Scientist, BGI Research APAC "]},"Case studies illustrating the transformative use of DCS Cloud in \n        projects such as the China Kadoorie Biobank (CKB) and 3D spatial brain mapping."]},{"name":"li","attribs":{},"children":["Hands-On Training: DCS Cloud for Single-Cell and Spatial Data Analysis – Part 1 (40 min) ",{"name":"em","attribs":{},"children":["Melvin Xiaochun Li, Field Bioinformatics Scientist, MGI APAC "]},"A guided walkthrough of the platform’s code-free analytical workflows,\n        interactive visualization tools, and AI-driven automation features for \n        single-cell and spatial data."]}]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Single-Cell Omics"]},{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]},{"name":"li","attribs":{},"children":["AI and Machine Learning in Genomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Streamline spatial and transcriptomic analysis with tidyomics"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","William Hutchison",{"name":"sup","attribs":{},"children":["1,2"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Walter and Eliza Hall Institute of Medical Research, Parkville, Australia"]},{"name":"li","attribs":{},"children":["Department of Medical Biology, University of Melbourne, Parkville, Australia"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 01:30PM"," to ","03:00PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["The tidyomics software ecosystem aims to bring the tidy R paradigm to \n        BioConductor. This workshop will introduce participants to tidyomics, \n        focusing on the tidySpatialExperiment and tidySingleCellExperiment \n        packages. We will begin with a brief overview of the \n        SingleCellExperiment class, the SpatialExperiment class and the \n        tidyverse ecosystem. We will then examine how tidyomics can enable tidy \n        omics data manipulation and plotting."]},{"name":"p","attribs":{},"children":["We will conclude by exploring additional utilities offered by \n        tidyomics, which aim to simplify complex and interesting analysis tasks. \n        Through this workshop, participants will be empowered to streamline \n        their own spatial and transcriptomic workflows."]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["A taster of in situ spatial transcriptomics experimental analysis"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Sarah Williams",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Magda Antczak",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Nick Matigan",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["QCIF"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 03:30PM"," to ","05:00PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n      This workshop aims to provide a practical introduction to using spatial \n      omics analysis to answer research questions; specifically, how to tackle \n      a comparative analysesbetween groups.\n      "]},{"name":"p","attribs":{},"children":["\n      We will start with a pre-processed in situ spatial dataset and undertake \n      some basic differential expression, look at differences of proportions of \n      celltypes, and exploratory plotting. We will work in R, using mostly \n      Bioconductor tools. We will use a cosMx dataset, but approaches would be \n      applicable to other technologies like Xenium or vizgen.\n      "]},{"name":"p","attribs":{},"children":["\n      Along the way we will discuss considerations specific to spatial analyses, \n      how to scale it up to a full analysis, and how to assess our results. We \n      will also introduce spatial sampler as a resource for examples and codes \n      snippets for analyses on spatial datasets.\n      "]},{"name":"p","attribs":{},"children":["\n      This workshop is aimed at newcomers to spatial analyses. Basic R skills \n      are required, a familiarity with single cell analysis is helpful but not essential.\n      "]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Spatial clustering with ClustSIGNAL – a spatial transcriptomics data analysis method"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Pratibha Panwar",{"name":"sup","attribs":{},"children":["1,2,3"]}]},{"name":"Fragment","attribs":[],"children":[", ","Shila Ghazanfar",{"name":"sup","attribs":{},"children":["1,2,3"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["School of Mathematics and Statistics, University of Sydney"]},{"name":"li","attribs":{},"children":["Sydney Precision Data Science Centre, University of Sydney"]},{"name":"li","attribs":{},"children":["Charles Perkins Centre, University of Sydney"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Thursday 27 November 2025, 03:30PM"," to ","05:00PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["\n        ClustSIGNAL is cell-type spatial clustering method for high-resolution \n        spatial transcriptomics data. It uses both the gene expression and \n        spatial locations of cells to group them into clusters. \n        ClustSIGNAL aims to:\n        "]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["overcome data sparsity using an adaptive smoothing approach that \n          uses cell-specific weights based on the heterogeneity level (entropy) \n          of a cell’s neighbourhood and"]},{"name":"li","attribs":{},"children":["embed spatial context information into the gene expression \n          generating a transformed, adaptively smoothed expression matrix that \n          is used for clustering."]}]},{"name":"p","attribs":{},"children":["\n          In this workshop, we will discuss how ClustSIGNAL works as a spatial \n          clustering method, and how it can be run on R. We will also cover \n          various parameters for ClustSIGNAL, assess cluster relevance through \n          accuracy metrics and marker genes, and explore entropy and \n          neighbourhood data generated by ClustSIGNAL.\n        "]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Introduction to spatial transcriptomics data analysis"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Dharmesh Bhuva",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Jiadong Mao",{"name":"sup","attribs":{},"children":["2"]}]},{"name":"Fragment","attribs":[],"children":[", ","Melody Jin",{"name":"sup","attribs":{},"children":["3"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Frazer Institute, University of Queensland"]},{"name":"li","attribs":{},"children":["Melbourne Integrative Genomics (MIG), University of Melbourne"]},{"name":"li","attribs":{},"children":["Walter and Eliza Hall Institute of Medical Research"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Friday 28 November 2025, 09:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 3 hour workshop with a 30 minute break"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["Spatial transcriptomics (ST) has enabled us to examine transcriptional \n      activities of cells in complex tissue microenvironments. However, current \n      workflows for ST data often involve fragmented pipelines with varying \n      normalisation strategies, cell type identification methods and \n      differential expression analyses. Our workshop addresses this critical \n      need by leveraging latest bioinformatic method development for ST data:"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Decoding cell states with Φ-Space"]},{"name":"li","attribs":{},"children":["Identify spatial differentially expressed genes with jazzPanda"]}]},{"name":"p","attribs":{},"children":["By bringing together experts and practitioners, we aim to move towards \n        consensus on best practices that balance analytical rigour with \n        biological interpretability, ultimately accelerating discoveries across\n        developmental biology, disease pathology and therapeutic development."]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Decoding Biological Data with Random Matrix Theory: Applications in Single-Cell Transcriptomics and Viral Vaccine Design"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Syed Faraz Ahmed",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Ahmed Abdul Quadeer",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Raymond Louie",{"name":"sup","attribs":{},"children":["2"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Department of Electrical and Electronic Engineering, The University of Melbourne"]},{"name":"li","attribs":{},"children":["School of Computer Science and Engineering, University of New South Wales"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Friday 28 November 2025, 09:00AM"," to ","10:30AM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["This workshop introduces participants to Random Matrix Theory (RMT) as a \n      powerful tool for denoising biological data and uncovering meaningful \n      structure in noisy covariance matrices, which is applicable across a wide \n      range of biological datasets. While examples will focus on single-cell \n      RNA-sequencing (scRNA-seq) and viral sequence data, the techniques are \n      broadly relevant to any domain where estimating correlation or \n      co-variation matters, including bulk transcriptomics, proteomics, and \n      microbiome studies."]},{"name":"p","attribs":{},"children":["Participants will learn how to integrate RMT into standard workflows, \n        such as identifying gene signatures of biological processes from \n        scRNA-seq and discovering mutationally vulnerable residues in viral \n        proteins to design efficient vaccines. Through an accessible overview, \n        live coding in R, and hands-on analysis of real-world datasets, \n        attendees will gain practical experience in spectral denoising and \n        inherent structure detection. By workshop's end, attendees will be \n        confident to apply these generalisable denoising methods to their own\n        research problems."]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Single-Cell Omics"]},{"name":"li","attribs":{},"children":["AI and Machine Learning in Genomics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["ATAClone: cancer clone identification and copy number estimation from single-cell ATAC-seq"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Lachlan Cain",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Sanduni Rajapaksa",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Peter MacCallum Cancer Centre"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Friday 28 November 2025, 09:00AM"," to ","10:30AM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["Single-cell analyses of cancer typically begin by identifying distinct \n        populations of cancer cells by unsupervised clustering. However, in \n        many cases this clustering is explained simply by differences in the \n        copy number. To detect and estimate these differences in copy number, \n        we have developed ATAClone. Applicable to both standalone and multiome \n        scATAC-seq assays, ATAClone first identifies cancer ‘clones’ with \n        shared copy number profiles, then estimates the copy number of each \n        clone jointly. Advantages over other single-cell copy number calling \n        tools include automated clone identification as well as ploidy-aware \n        copy number estimation. In this workshop, participants will be guided \n        step-by-step through a typical ATAClone workflow including key \n        considerations at each analysis stage, as well as examples of \n        downstream analyses using ATAClone outputs."]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Single-Cell Omics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Advanced Marker Identification and Data Analysis with mastR and smartid"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Jinjin Chen",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Chin Wee Tan",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Liyang Fei",{"name":"sup","attribs":{},"children":["2"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Bioinformatics Division of Walter and Eliza Hall Institute of Medical Research"]},{"name":"li","attribs":{},"children":["Peter MacCallum Cancer Centre"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Friday 28 November 2025, 11:00AM"," to ","12:30PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["This workshop is designed for users who want to learn how to use mastR \n        and smartid for biomarker discovery on transcriptomic data. mastR \n        (Chen et al Bioinformatics 2025, 41(3):btaf114) is an R package \n        enabling automated marker screening across multiple groups, combine \n        results from different differential expression comparisons, and apply \n        filters to reduce background noise from unrelated tissues. With smartid, \n        we’ll explore its TF‐IDF‐based approach to ranking markers, which is \n        particularly useful for identifying genes linked to small or rare \n        populations from imbalanced data."]},{"name":"p","attribs":{},"children":["This workshop will introduce the key workflows of the two packages, \n        look at practical tips, common pitfalls, including a few examples for \n        using of these tools. By the end of the session, participants should be \n        able to run both packages confidently and know how to integrate their \n        results into a broader analysis workflow. (mastr: ",{"name":"a","attribs":{"href":"https://bioconductor.org/packages/release/bioc/html/mastR.html"},"children":["https://bioconductor.org/packages/release/bioc/html/mastR.html"]}," & smartid: ",{"name":"a","attribs":{"href":"https://bioconductor.org/packages/release/bioc/html/smartid.html"},"children":["https://bioconductor.org/packages/release/bioc/html/smartid.html"]},")"]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Single-Cell Omics"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Interpreting benchmarking results with BenchmarkInsights"]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Xiaoqi Cabiria Liang",{"name":"sup","attribs":{},"children":["1,2,3"]}]},{"name":"Fragment","attribs":[],"children":[", ","Dario Strbenac",{"name":"sup","attribs":{},"children":["1,2,3"]}]},{"name":"Fragment","attribs":[],"children":[", ","Yue Cao",{"name":"sup","attribs":{},"children":["1,2,3"]}]},{"name":"Fragment","attribs":[],"children":[", ","Jean Yang",{"name":"sup","attribs":{},"children":["1,2,3"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["School of Mathematics and Statistics, The University of Sydney, NSW 2006, Australia."]},{"name":"li","attribs":{},"children":["Sydney Precision Data Science Centre, The University of Sydney, NSW 2006, Australia"]},{"name":"li","attribs":{},"children":["Charles Perkins Centre, The University of Sydney, NSW 2006, Australia."]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Friday 28 November 2025, 01:30PM"," to ","03:00PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["Benchmarking studies are critical for assessing method performance. \n        This workshop is for researchers who want to learn different ways of \n        visualising and comparing method performance after running a \n        benchmarking study. It will use the benchmarkInsights module from our\n        R package BenchHub that provides a series of functions to visualise \n        benchmarking results. The session will begin with a brief background \n        on benchmarking metrics, followed by a hands-on session where \n        participants will:"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["interact with benchmarkInsights class"]},{"name":"li","attribs":{},"children":["generate summary visualisation"]},{"name":"li","attribs":{},"children":["inspect method performance across methods and datasets"]},{"name":"li","attribs":{},"children":["identify robust methods"]}]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Scalable Workflows & Compute Infrastructure"]}]}]}]},{"name":"div","attribs":{"className":"workshop-detail"},"children":[{"name":"div","attribs":{"className":"workshop-title"},"children":[{"name":"h2","attribs":{},"children":["Interrogating spatial transcriptomics: hands-on spatial domain, neighborhood & density analysis with DOMINO, hooscanR and scider."]}]},{"name":"div","attribs":{"className":"workshop-presenter"},"children":[{"name":"h3","attribs":{},"children":["Presenter(s)"]},{"name":"Fragment","attribs":[],"children":["","Ning Liu",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Nora Liu",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Monika Mohenska",{"name":"sup","attribs":{},"children":["1"]}]},{"name":"Fragment","attribs":[],"children":[", ","Jose Polo",{"name":"sup","attribs":{},"children":["1"]}]}]},{"name":"div","attribs":{"className":"workshop-institution"},"children":[{"name":"h3","attribs":{},"children":["Presenter Organisation(s)"]},{"name":"ol","attribs":{},"children":[{"name":"li","attribs":{},"children":["Adelaide Centre for Epigenetics, University of Adelaide"]}]}]},{"name":"div","attribs":{"className":"workshop-schedule"},"children":[{"name":"h3","attribs":{},"children":["Workshop Schedule"]},"Friday 28 November 2025, 01:30PM"," to ","03:00PM",{"name":"p","attribs":{},"children":["This will be a 90 minute workshop without breaks"]}]},{"name":"div","attribs":{"className":"workshop-abstract"},"children":[{"name":"h3","attribs":{},"children":["Workshop Outline"]},{"name":"body","attribs":{},"children":[{"name":"p","attribs":{},"children":["Spatial transcriptomics (ST) data offer molecule position with \n        quantitative gene expression in situ. Downstream analysis of ST data \n        can be tailored to answer different biological questions, from \n        identifying spatial domains, detecting spatial neighborhoods at \n        single-cell level to quantifying cell-type-specific density landscapes."]},{"name":"p","attribs":{},"children":["In this workshop, we introduce three complementary Python and \n        R/Bioconductor tools to tackle these questions. Through a guided \n        workflow, participants will learn how to preprocess ST data, including \n        performing quality control, data normalization and cell type annotation, \n        configure parameters for each method, visualise results and interpret \n        outputs in biological contexts such as tumor domains detection, cell \n        neighborhood heterogeneity comparison and tumor density characterization."]},{"name":"p","attribs":{},"children":["By the end of the workshop, attendees will be equipped to integrate \n        these methods into their own projects, allowing for reproducible, \n        data-driven insights about complex tissue architectures."]}]}]},{"name":"div","attribs":{"className":"workshop-stream"},"children":[{"name":"h3","attribs":{},"children":["Workshop Stream(s)"]},{"name":"ul","attribs":{},"children":[{"name":"li","attribs":{},"children":["Spatial Transcriptomics"]}]}]}]}]},{"id":"Day","name":"Day","type":["ordered","factor"],"filterInput":{"name":"select","attribs":{"aria-label":"Filter Day","style":{"width":"100%","height":"28px"},"onChange":"function(_e){(function(event){Reactable.setFilter('tbl-workshop-schedule', 'Day', event.target.value || undefined)}).apply(event.target,[_e])}"},"children":[{"name":"option","attribs":{"value":""},"children":["All"]},{"name":"option","attribs":{},"children":["Thursday"]},{"name":"option","attribs":{},"children":["Friday"]}]},"maxWidth":80},{"id":"Start","name":"Start","type":"Date","cell":["09:00","09:00","09:00","09:00","09:00","11:00","13:30","13:30","13:30","13:30","15:30","15:30","09:00","09:00","09:00","11:00","13:30","13:30"],"maxWidth":65},{"id":"End","name":"End","type":"Date","cell":["12:30","12:30","12:30","12:30","10:30","12:30","17:00","17:00","15:00","15:00","17:00","17:00","12:30","10:30","10:30","12:30","15:00","15:00"],"maxWidth":65},{"id":"Domain","name":"Domain","type":"character","filterInput":{"name":"select","attribs":{"aria-label":"Filter Domain","style":{"width":"100%","height":"28px"},"onChange":"function(_e){(function(event){Reactable.setFilter('tbl-workshop-schedule', 'Domain', event.target.value || undefined)}).apply(event.target,[_e])}"},"children":[{"name":"option","attribs":{"value":""},"children":["All"]},{"name":"option","attribs":{},"children":["ABACBS"]},{"name":"option","attribs":{},"children":["BiocAsia"]}]},"maxWidth":80},{"id":"Stream","name":"Stream/Theme","type":"list","cell":["Professional Bioinformatics<br>AI and Machine Learning in Genomics<br>Production Deployments","Scalable Workflows & Compute Infrastructure","Metagenomics & Microbiome analyses","Spatial Transcriptomics","Long Read Technology","Long Read Technology","Scalable Workflows & Compute Infrastructure<br>Professional Bioinformatics","Metagenomics & Microbiome analyses","Single-Cell Omics<br>Spatial Transcriptomics<br>AI and Machine Learning in Genomics","Spatial Transcriptomics","Spatial Transcriptomics","Spatial Transcriptomics","Spatial Transcriptomics","Single-Cell Omics<br>AI and Machine Learning in Genomics","Single-Cell Omics","Single-Cell Omics","Scalable Workflows & Compute Infrastructure","Spatial Transcriptomics"],"filterInput":{"name":"select","attribs":{"aria-label":"Filter Stream","style":{"width":"100%","height":"28px"},"onChange":"function(_e){(function(event){Reactable.setFilter('tbl-workshop-schedule', 'Stream', event.target.value || undefined)}).apply(event.target,[_e])}"},"children":[{"name":"option","attribs":{"value":""},"children":["All"]},{"name":"option","attribs":{},"children":["AI and Machine Learning in Genomics"]},{"name":"option","attribs":{},"children":["Long Read Technology"]},{"name":"option","attribs":{},"children":["Metagenomics & Microbiome analyses"]},{"name":"option","attribs":{},"children":["Production Deployments"]},{"name":"option","attribs":{},"children":["Professional Bioinformatics"]},{"name":"option","attribs":{},"children":["Scalable Workflows & Compute Infrastructure"]},{"name":"option","attribs":{},"children":["Single-Cell Omics"]},{"name":"option","attribs":{},"children":["Spatial Transcriptomics"]}]},"html":true},{"id":"Title","name":"Workshop Title","type":"character"},{"id":"Presenter","name":"Presenter(s)","type":"list","cell":["Raisa Hashem<br>Cain Hopwood<br>Emma Arrigo","Johan Gustafsson<br>Ziad Al Bkhetan<br>Thomas Litfin<br>Sarah Beecroft<br>Georgie Samaha<br>Farah Zaib Khan","Tuomas Borman<br>Leo Lahti","Daniel Kim<br>Farhan Ameen<br>Lijia Yu<br>Shreya Rajesh Rao<br>Ellis Patrick<br>Jean Yang","Andre Sim","Andre Sim","Wytamma Wirth<br>Andrew Lonsdale<br>Leo Featherstone<br>Torsten Seemann","Feargal Ryan<br>Calum Walsh<br>Geraldine Kong<br>Vini Salazar<br>Don Teng","Karthik Pullela<br>Mingyan Fang<br>Melvin Xiaochun Li","William Hutchison","Sarah Williams<br>Magda Antczak<br>Nick Matigan","Pratibha Panwar<br>Shila Ghazanfar","Dharmesh Bhuva<br>Jiadong Mao<br>Melody Jin","Syed Faraz Ahmed<br>Ahmed Abdul Quadeer<br>Raymond Louie","Lachlan Cain<br>Sanduni Rajapaksa","Jinjin Chen<br>Chin Wee Tan<br>Liyang Fei","Xiaoqi Cabiria Liang<br>Dario Strbenac<br>Yue Cao<br>Jean Yang","Ning Liu<br>Nora Liu<br>Monika Mohenska<br>Jose Polo"],"html":true,"maxWidth":140},{"id":"Institution","name":"Institution(s)","type":"list","show":false,"filterInput":{"name":"select","attribs":{"aria-label":"Filter Institution","style":{"width":"100%","height":"28px"},"onChange":"function(_e){(function(event){Reactable.setFilter('tbl-workshop-schedule', 'Institution', event.target.value || undefined)}).apply(event.target,[_e])}"},"children":[{"name":"option","attribs":{"value":""},"children":["All"]},{"name":"option","attribs":{},"children":[" Parkville"]},{"name":"option","attribs":{},"children":["3000"]},{"name":"option","attribs":{},"children":["3010"]},{"name":"option","attribs":{},"children":["3052"]},{"name":"option","attribs":{},"children":["Adelaide Centre for Epigenetics"]},{"name":"option","attribs":{},"children":["Amazon Web Services"]},{"name":"option","attribs":{},"children":["Australia"]},{"name":"option","attribs":{},"children":["Australia."]},{"name":"option","attribs":{},"children":["Australian BioCommons"]},{"name":"option","attribs":{},"children":["Bioinformatics Division of Walter and Eliza Hall Institute of Medical Research"]},{"name":"option","attribs":{},"children":["Centre for Pathogen Genomics"]},{"name":"option","attribs":{},"children":["Charles Perkins Centre"]},{"name":"option","attribs":{},"children":["Children’s Medical Research Institute"]},{"name":"option","attribs":{},"children":["Computational Systems Biology Unit"]},{"name":"option","attribs":{},"children":["Department of Electrical and Electronic Engineering"]},{"name":"option","attribs":{},"children":["Department of Medical Biology"]},{"name":"option","attribs":{},"children":["Department of Microbiology and Immunology"]},{"name":"option","attribs":{},"children":["Faculty of Medicine and Health"]},{"name":"option","attribs":{},"children":["Finland"]},{"name":"option","attribs":{},"children":["Flinders University"]},{"name":"option","attribs":{},"children":["Frazer Institute"]},{"name":"option","attribs":{},"children":["Kirby Institute"]},{"name":"option","attribs":{},"children":["Melbourne"]},{"name":"option","attribs":{},"children":["Melbourne Bioinformatics"]},{"name":"option","attribs":{},"children":["Melbourne Integrative Genomics (MIG)"]},{"name":"option","attribs":{},"children":["Metabolomics SA"]},{"name":"option","attribs":{},"children":["MGI"]},{"name":"option","attribs":{},"children":["Murdoch Children’s Research Institute"]},{"name":"option","attribs":{},"children":["NSW"]},{"name":"option","attribs":{},"children":["NSW 2006"]},{"name":"option","attribs":{},"children":["Oxford Nanopore Technologies"]},{"name":"option","attribs":{},"children":["Parkville"]},{"name":"option","attribs":{},"children":["Pawsey Supercomputing Research Centre"]},{"name":"option","attribs":{},"children":["Peter MacCallum Cancer Centre"]},{"name":"option","attribs":{},"children":["QCIF"]},{"name":"option","attribs":{},"children":["School of Computer Science and Engineering"]},{"name":"option","attribs":{},"children":["School of Mathematics and Statistics"]},{"name":"option","attribs":{},"children":["School of Medical Sciences"]},{"name":"option","attribs":{},"children":["Sir Peter MacCallum Department of Oncology"]},{"name":"option","attribs":{},"children":["Sydney"]},{"name":"option","attribs":{},"children":["Sydney Informatics Hub"]},{"name":"option","attribs":{},"children":["Sydney Precision Data Science Centre"]},{"name":"option","attribs":{},"children":["The Peter Doherty Institute for Infection and Immunity"]},{"name":"option","attribs":{},"children":["The University of Melbourne"]},{"name":"option","attribs":{},"children":["The University of Sydney"]},{"name":"option","attribs":{},"children":["University of Adelaide"]},{"name":"option","attribs":{},"children":["University of Melbourne"]},{"name":"option","attribs":{},"children":["University of New South Wales"]},{"name":"option","attribs":{},"children":["University of Queensland"]},{"name":"option","attribs":{},"children":["University of Sydney"]},{"name":"option","attribs":{},"children":["University of Turku"]},{"name":"option","attribs":{},"children":["UNSW"]},{"name":"option","attribs":{},"children":["Victoria"]},{"name":"option","attribs":{},"children":["Walter and Eliza Hall Institute of Medical Research"]},{"name":"option","attribs":{},"children":["Westmead"]}]}}],"filterable":true,"searchable":true,"pagination":false,"theme":{"style":{"fontSize":14}},"elementId":"tbl-workshop-schedule","dataKey":"2dd9d0af2201fd35fe8fff4fd40180d2"},"children":[]},"class":"reactR_markup"},"evals":["tag.attribs.columns.1.filterInput.attribs.onChange","tag.attribs.columns.4.filterInput.attribs.onChange","tag.attribs.columns.5.filterInput.attribs.onChange","tag.attribs.columns.8.filterInput.attribs.onChange"],"jsHooks":[]}</script>
</div>
</figure>
</div>
<p><em>The workshop organisers gratefully thank our sponsors</em></p>
<p><img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/db60bf70-5296-4008-a65a-7ff416c21d05/pawsey-logo-beige.png?format=2500w" class="img-fluid"> <img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/b63a95b5-032a-4d09-8fa9-3e1a6931a1ad/Screenshot+2025-07-28+at+14.25.24.png?format=500w" class="img-fluid"> <img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/22378394-6648-41f8-be38-82636e29dc1d/Screenshot+2025-07-02+at+1.52.27%E2%80%AFpm.png?format=1200w" class="img-fluid"> <img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/e5b29f2c-e43c-438c-9189-0928a78f1c19/Bickfords_Australia_logo.webp?format=400w" class="img-fluid"> <img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/000809d4-ab00-4aa8-bbb9-89f41a7925dd/Screenshot+2025-07-02+at+2.06.13%E2%80%AFpm.png?format=1000w" class="img-fluid"> <img src="https://images.squarespace-cdn.com/content/v1/5423875be4b03f0c482a58c4/7ecf5273-876c-4f3c-bc76-364438cb6f5d/Screenshot+2025-07-02+at+2.06.08%E2%80%AFpm.png?format=1000w" class="img-fluid"></p>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>